Add note about some cameras needing to seek to get a valid snapshot (#27542)
home-assistant/home-assistant.github.io: source/_integrations/onvif.markdown

| Extra FFmpeg arguments | Extra options to pass to `ffmpeg`, e.g., image quality or video filter options. More details in [`ffmpeg` integration](/integrations/ffmpeg). |
| Use wallclock as timestamps | ([Advanced Mode](/blog/2019/07/17/release-96/#advanced-mode) only) Rewrite the camera timestamps. This may help with playback or crashing issues from Wi-Fi cameras or cameras of certain brands (e.g., EZVIZ). |

#### Snapshots

> Some cameras will not produce usable snapshots with larger stream sizes and may require seeking in the stream to get a usable snapshot. Setting the `Extra FFmpeg arguments` to `-pred 1 -ss 00:00:05 -frames:v 1` will cause the snapshot to be taken 5 seconds into the stream.

Author: ```suggestion
Some cameras will not produce usable snapshots with larger stream sizes.

By default the integration will only enable the camera entity for the first H264 profile. If you are unable to get working snapshot:

- If there are additional camera entities available for other profiles, try enabling those entities.
- Set the `Extra FFmpeg arguments` to `-pred 1 -ss 00:00:05 -frames:v 1` to cause the snapshot to be taken 5 seconds into the stream.
```

| Extra FFmpeg arguments | Extra options to pass to `ffmpeg`, e.g., image quality or video filter options. More details in [`ffmpeg` integration](/integrations/ffmpeg). |
| Use wallclock as timestamps | ([Advanced Mode](/blog/2019/07/17/release-96/#advanced-mode) only) Rewrite the camera timestamps. This may help with playback or crashing issues from Wi-Fi cameras or cameras of certain brands (e.g., EZVIZ). |

#### Snapshots

Some cameras will not produce usable snapshots with larger stream sizes.

> By default the integration will only enable the camera entity for the first H264 profile. If you are unable to get working snapshot:

Author: ```suggestion
By default, the integration will only enable the camera entity for the first H264 profile. If you are unable to get a working snapshot:
```

> - If there are additional camera entities available for other profiles, try enabling those entities.

Author: ```suggestion
- If additional camera entities are available for other profiles, try enabling those entities.
```
